src/main/java/net/snowflake/client/jdbc/SnowflakeFileTransferConfig.java

```diff
@@ -83,6 +87,14 @@ public boolean getUseS3RegionalUrl() {
     return useS3RegionalUrl;
   }
 
+  public String getStreamingIngestClientName() {
+    return this.streamingIngestClientName;
+  }
+
+  public String getStreamingIngestClientKey() {
+    return this.streamingIngestClientKey;
+  }
```

Comment:
I think we might want to document these somewhere since it's not immediately obvious what they are.